src/core/default_appearance.js

```diff
@@ -87,6 +87,94 @@ function parseDefaultAppearance(str) {
   return new DefaultAppearanceEvaluator(str).parse();
 }
 
+class AppearanceEvaluator extends EvaluatorPreprocessor {
```

Comment:
To reduce confusion with the pre-existing class, please use a slightly more specific name:
```suggestion
class AppearanceStreamEvaluator extends EvaluatorPreprocessor {
```